package.json

```diff
@@ -32,6 +32,7 @@
   "dependencies": {
     "@kaizen/design-tokens": "^10.0.11",
     "@kaizen/tailwind": "*",
+    "@storybook/blocks": "^7.0.0-beta.64",
```

Comment:
This comes with `@storybook` so you don't need to import it.

Reply:
If I don't have it as a dependency I get [this eslint](https://github.com/cultureamp/kaizen-design-system/pull/3359#discussion_r1131905717) issue I mentioned

Reply:
@gyfchong we can always move it back for now and then address this later if it is an issue - though I do wonder about the next release if the next alpha was anything to go by

Reply:
Yeah put the Unstyled back to Preview and move on lol